test/lit/passes/trace-calls.wast

```diff
@@ -0,0 +1,124 @@
+;; RUN: foreach %s %t wasm-opt --enable-simd --trace-calls="noparamsnoresults;singleparamnoresults,f64;multiparamsnoresults:tracempnr,i32,i64,f32;noparamssingleresult,v128;multiparamssingleresult,v128,i32,v128" -S -o - | filecheck %s
```

Comment:
There should be a line here like this:
```
;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
```
That allows the auto-update script to handle it. After adding that run `./scripts/update_lit_checks.py test/lit/passes/trace-calls.wast`